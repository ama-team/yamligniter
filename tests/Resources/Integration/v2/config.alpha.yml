base_url: 'http://codeigniter.com/'
log_threshold: 2
global_xss_filtering: true
custom_config_option: 12
