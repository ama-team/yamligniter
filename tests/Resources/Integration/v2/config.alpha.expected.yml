config:
  base_url: 'http://codeigniter.com/'
  index_page: 'index.php'
  uri_protocol: 'AUTO'
  url_suffix: ''
  language: 'english'
  charset: 'UTF-8'
  enable_hooks: false
  subclass_prefix: 'MY_'
  permitted_uri_chars: 'a-z 0-9~%.:_\-'
  allow_get_array: true
  enable_query_strings: false
  controller_trigger: 'c'
  function_trigger: 'm'
  directory_trigger: 'd'
  log_threshold: 2
  log_path: ''
  log_date_format: 'Y-m-d H:i:s'
  cache_path: ''
  encryption_key: ''
  sess_cookie_name: 'ci_session'
  sess_expiration: 7200
  sess_expire_on_close: false
  sess_encrypt_cookie: false
  sess_use_database: false
  sess_table_name: 'ci_sessions'
  sess_match_ip: false
  sess_match_useragent: true
  sess_time_to_update: 300
  cookie_prefix: ''
  cookie_domain: ''
  cookie_path: '/'
  cookie_secure: false
  global_xss_filtering: true
  csrf_protection: false
  csrf_token_name: 'csrf_test_name'
  csrf_cookie_name: 'csrf_cookie_name'
  csrf_expire: 7200
  compress_output: false
  time_reference: 'local'
  rewrite_short_tags: false
  proxy_ips: ''
  custom_config_option: 12
